docs/integrations/sources/mssql.md

```diff
@@ -59,10 +62,140 @@ _Coming soon: suggestions on how to create this user._
 
 Your database user should now be ready for use with Airbyte.
 
+## Change Data Capture : CDC
+
+We use [SQL Server's change data capture feature](https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-2017)
+to capture row-level `INSERT`, `UPDATE` and `DELETE` operations that occur on cdc-enabled tables.
+
+Some extra setup requiring at least *db_owner* permissions on the database(s) you intend to sync from will be required (detailed [below](mssql.md#Setting-up-CDC-for-MSSQL)).
+
+Please read the [CDC docs](../../understanding-airbyte/cdc.md) for an overview of how Airbyte approaches CDC.
+
+### Should I use CDC for MSSQL?
+
+* If you need a record of deletions and can accept the limitations posted below, CDC is the way to go!
+* If your data set is small and/or you just want a snapshot of your table in the destination, consider using Full Refresh replication for your table instead of CDC.
+* If the limitations below prevent you from using CDC and your goal is to maintain a snapshot of your table in the destination, consider using non-CDC incremental and occasionally reset the data and re-sync.
+* If your table has a primary key but doesn't have a reasonable cursor field for incremental syncing \(i.e. `updated_at`\), CDC allows you to sync your table incrementally.
+
+### CDC Limitations
+
+* Make sure to read our [CDC docs](../../understanding-airbyte/cdc.md) to see limitations that impact all databases using CDC replication.
```

Comment:
I don't see why not since MSSQL handles CDC individually on a table by table basis, however I haven't tested it. Worth including with work on that issue.